crates/store/re_tf/src/transform_resolution_cache.rs

```diff
@@ -828,11 +835,18 @@ impl TransformResolutionCache {
                 // We now look only at the times in the time column that are relevant for this child-frame.
                 // Note that there may be more times than actual relevant updates, but crucially, all queries
                 // to the current entity path yield information about the sources in `source_frames`.
-                let times_with_potential_update = time_column
-                    .times()
-                    // TODO(andreas): For sorted time columns we could speed this up a bit.
-                    .filter(|time| time_range.contains(time))
-                    .collect_vec();
+                let times_with_potential_update = if time_column.time_range().min
+                    >= time_range.start
+                    && time_column.time_range().max < time_range.end
+                {
+                    time_column.times().collect_vec()
+                } else {
+                    time_column
+                        .times()
+                        // TODO(andreas): For sorted time columns we could speed this up a bit.
+                        .filter(|time| time_range.contains(time))
+                        .collect_vec()
```

Comment:
If you replace `.times()` with `.times_raw()` (in both branches), you could completely avoid allocations in the first branch (e.g. using a `let times_with_potential_update: Cow<[i64]>`).

Reply:
then I have to swap over to i64 everywhere instead of `TimeInt`. But that sounds worth it. Thanks!

Reply:
was trivial and gives 5.5% perf on `build_from_entity_db`. Also just feels good :D